draft-ietf-rtcweb-jsep.xml

```diff
@@ -741,66 +741,80 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
-          satisfied and the sender MUST NOT transmit the track.</t>
-
-          <t>In the special case of receiving a maximum resolution of
-          [0, 0], as described above, the sender MUST NOT transmit the
-          track.</t>
+          <t>This specification prescribes more specific behavior.
+          When a MediaStreamTrack, which is producing
+          video of a certain resolution (the "track resolution"), is attached
+          to a RtpSender, which is encoding the track video
+          at the same or lower resolution(s) (the "encoder resolutions"),
+          and a remote description is applied that references the sender and
+          contains valid "a=imageattr recv" attributes, it will
+          follow the rules below to ensure the sender does not transmit a
+          resolution that would exceed the size criteria specified in the
+          attributes. These rules are followed
+          as long as the attributes remain present in the remote description,
+          including if the track changes its resolution,
+          or if the track is replaced with a different track.
+          </t>
+
+          <t>Depending on how the RtpSender is configured, it may be producing
+          a single resolution, or, if simulcast <xref target="sec.simulcast" />
+          has been negotiated, multiple specific resolutions. In addition,
+          depending on the configuration, each encoding may have the
+          flexibility to reduce resolution when needed, or may be locked to
+          a specific output resolution.
+          </t>
+
+          <t>For each encoding being produced by the RtpSender, the following
+          rules are applied to determine what should be transmitted:
+            <list style="symbols">
+              <t>First, the right "a=imageattr recv" attribute is selected.
+              While JSEP endpoints will not include more than one such
+              attribute per media format, non-JSEP endpoints may send
+              multiple relevant "a=imageattr recv" attributes in a m=
+              section. The correct attribute is selected by taking the
+              attribute with the highest "q=" value from the set of attributes
+              that reference the media format that has been selected for the
+              specified encoding. If multiple attributes have the same "q="
+              value, the one that appears first in the m= section is used.
+              </t>
+
+              <t>If there is an applicable "a=imageattr recv" attribute for
+              the encoding, the limits from the attribute are then compared to
+              the encoder resolution. Only the specific limits mentioned below
+              are considered; any other values, e.g. picture aspect ratio,
+              MUST be ignored.
+              Note that when considering a MediaStreamTrack that is producing
+              rotated video, the unrotated resolution MUST be used for the
+              checks. This is
```

Comment:
I tried that but the sentence ended up feeling run-on:

              Note that when considering a MediaStreamTrack that is producing
              rotated video, the unrotated resolution MUST be used for the
              checks, regardless of whether the receiver supports
              performing receive-side rotation (e.g., through CVO
              [TS26.114]), as it significantly simplifies
              the matching logic.